### ISSUE TITLE
[Bug]: Supply Convoys stacking on one city (Anizay)

### ISSUE THREAD

**Issue by Ghost1391a** [official-antistasi-community/A3-Antistasi#3280]
### Describe the bug

I was in a mission with friends and we are getting constant supply convoys to one city that all happen minutes apart from each other. Its basically impossible to attack all of them that come at the same time, and is resulting in taking cities to be impossible. 
Strangely this is only happening in Anizay. 

### How to reproduce

![20240609235930_1](https://github.com/official-antistasi-community/A3-Antistasi/assets/172236117/a4238aef-901c-4a76-a3d0-d441b9364f38)
![20240609235931_1](https://github.com/official-antistasi-community/A3-Antistasi/assets/172236117/86bc3896-41aa-4213-80c3-4a1b7eac8377)
![20240609235932_1](https://github.com/official-antistasi-community/A3-Antistasi/assets/172236117/15c8f58c-1760-428e-af7e-3b436f72f695)
![20240609235933_1](https://github.com/official-antistasi-community/A3-Antistasi/assets/172236117/4595002a-9912-4bd9-ad3e-a65924ee0764)

### Version

Version 3.5.4

### Have you altered the code?

No

### Map

Anizay

### What server?

Private dedicated server

### Time bug occured (Server time/UTC)

0:10 EST

### Mods

```html
RHSAFRF	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=843425103
RHSUSAF	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=843577117
CBA_A3	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=450814997
RHSGREF	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=843593391
RHSSAF	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=843632231
CUP Weapons	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=497660133
Enhanced Movement	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=333310405
Enhanced Movement Rework	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=2034363662
WMO - Walkable Moving Objects	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=925018569
CUP Terrains - Core	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=583496184
CUP Terrains - Maps	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=583544987
Antistasi - The Mod	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=2867537125
Anizay	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=1537973181
ace	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=463939057
3CB Factions	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=1673456286
Loot to Vehicle for ACE and Antistasi	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=1439779114
DUI - Squad Radar	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=1638341685
ACE 3 Extension (Animations and Actions)	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=766491311
CUP Units	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=497661914
CUP Vehicles	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=541888371
```

**Comment by jaj22** [comment]
Can you show me a zoomed-out picture of the map? Need to know what choices the AI has.

**Comment by Ghost1391a** [comment]
![20240610002134_1](https://github.com/official-antistasi-community/A3-Antistasi/assets/172236117/3fd74be9-91d6-4222-8735-ad784475a7bc)

**Comment by Ghost1391a** [comment]
We lost Gorqan, because of the multiple supply convoys actually.

**Comment by Ghost1391a** [comment]
We also seem to get more again, to Mian, and its the same again. 
![20240610002409_1](https://github.com/official-antistasi-community/A3-Antistasi/assets/172236117/2abd5f02-b554-4c4d-ac2e-4b6a4a0bfdaf)
![20240610002410_1](https://github.com/official-antistasi-community/A3-Antistasi/assets/172236117/5189b996-ce49-4bb1-8ba1-00b9336d3811)

**Comment by jaj22** [comment]
I'm gonna guess you're playing reb vs gov, so the enemy has no choice but to send supply convoys. They're quite cheap in the early game so they could sustain maybe one per 5-10min if you don't destroy them. If you do destroy them, it'll be ~4x slower, plus whatever the mission penalty was for those.

I already put an additional "planning cost" on supply convoys in #3273 so that should make them much less spammy.

**Comment by jaj22** [comment]
Should be fixed by #3273
